static/js/errorlogs/viewlogs.js

```diff
@@ -1,4 +1,20 @@
 MEDUSA.errorlogs.viewlogs = function() {
+    if (MEDUSA.config.fanartBackground) {
```

Comment:
All good, just try not to duplicate code. In Python, mako and js we have common files so things like this don't happen otherwise you'll edit one to change something and not the others and it just fucks a lot up. We've learnt that from old Sickrage code..